project/DottySupport.scala

```diff
@@ -12,8 +12,8 @@ import sbt.librarymanagement.{
   * Settings to support validation of TastyUnpickler against the release of dotty with the matching TASTy version
   */
 object TastySupport {
-  val supportedTASTyRelease = "3.0.0-M2" // TASTy version 25.1
-  val scala3Compiler = "org.scala-lang" % "scala3-compiler_3.0.0-M2" % supportedTASTyRelease
+  val supportedTASTyRelease = "3.0.0-M3-bin-20201215-cbe50b3-NIGHTLY" // TASTy version 26.1
```

Comment:
Perhaps we can wait one more day to be able to use 3.0.0-M3 here ;)

Reply:
```suggestion
  val supportedTASTyRelease = "3.0.0-M3" // TASTy version 26.1
```

Reply:
done